src/mono/mono/mini/interp/transform.c

```diff
@@ -3939,7 +3941,8 @@ recursively_make_pred_seq_points (TransformData *td, InterpBasicBlock *bb)
 {
 	SeqPoint ** const MONO_SEQ_SEEN_LOOP = (SeqPoint**)GINT_TO_POINTER(-1);
 
-	GArray *predecessors = g_array_new (FALSE, TRUE, sizeof (gpointer));
+	dn_vector_ptr_t predecessors = {0,};
```

Comment:
No need to init the dn_vector_ptr_t, it will be memset by dn_vector_ptr_init.

Reply:
I'm not sure if C compilers (or linters) would like that.